movieReviewApp/src/test/java/com/mobiquity/movieReviewApp/service/PasswordRecoverServiceImplTest.java

```diff
@@ -40,23 +50,21 @@ public void checkIfActivationLinkForPasswordIsSent() {
     assertEquals("Password Reset link sent to your email", result);
   }
 
+  // Test for Forgot password.
   @Test
+  @DataSet(value = "password.xml")
   public void checkIfNewPasswordIsUpdated() {
-    ResetPassword resetPassword = new ResetPassword();
-    resetPassword.setEmailId("ds@gmail.com");
-    //resetPassword.setOldPassword("asdfg");
-    resetPassword.setNewPassword("qwerty");
-    String result = passwordRecoverService.UpdatePassword(resetPassword);
+    ForgotPassword forgotPassword = new ForgotPassword();
+    forgotPassword.setToken(
+        "eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJhc2RmZ2hAZ21haWwuY29tIDEiLCJleHAiOjE1ODE1ODk2MjUsImlhdCI6MTU4MTUwMzIyNX0.yRREsDXalL-aIV2u3rXyHIgpx2NL4ZZ5bk4BwUyRcsCyD18YuZzOPx_sblMs059ZdYAsNIJWwDIfnqxEsqjjfA");
+    forgotPassword.setPassword("zxcvb");
+    String result = passwordRecoverService.UpdatePassword(forgotPassword);
     assertEquals("New Password is Updated", result);
-
   }
 
   @Test
   public void checkIfEmailIdRetrievedCorrectlyForForgotPasswordLink() {
```

Comment:
Can't really tell what this test is doing by the name or the internals of the method..

Reply:
This test has written for end point activationLinkForNewPassword. Right now we decided not expose this api. We can comment this test as well for now.